webapp/src/Service/ImportProblemService.php

```diff
@@ -819,6 +820,11 @@ public function importZippedProblem(
             $this->em->flush();
             $testcases = $problem->getTestcases()->toArray();
             if (count(array_filter($testcases, fn($t) => !$t->getDeleted())) == 0) {
+                // We need to reload the contest problem after the call(s) to the eventLogService.
```

Comment:
It is unclear to my (on first glance) why we have to do this in the `if` and not above.